aten/src/ATen/native/mkldnn/Normalization.cpp

```diff
@@ -48,7 +48,7 @@ namespace native {
 std::tuple<Tensor, Tensor, Tensor> mkldnn_layer_norm_last_index_weight_bias_f32(
     const Tensor& input,
     IntArrayRef normalized_shape, const Tensor& weight, const Tensor& bias,
-    double eps) {
+    double eps, bool inplace) {
```

Comment:
typically the way that inplace is handled is that there is just a `Tensor& result` input, and the callers are responsible for initializing it as a new tensor or as the input